RELEASES.md

```diff
@@ -0,0 +1,112 @@
+
+## Asterinas Kernel
+
+We have made the following key changes to the Asterinas kernel:
+
+* New system calls or features:
+    * Memory:
+        * [Add the `mremap` system call](https://github.com/asterinas/asterinas/pull/2162)
+        * [Add the `msync` system call based on an inefficient implementation](https://github.com/asterinas/asterinas/pull/2154)
+        * [Add the `memfd_create` system call](https://github.com/asterinas/asterinas/pull/2149)
+    * Processes and IPC:
+        * [Add the `pidfd_open` system call along with the `CLONE_PIDFD` flag](https://github.com/asterinas/asterinas/pull/2151)
+    * File systems and I/O in general:
+        * [Add the `close_range` system call](https://github.com/asterinas/asterinas/pull/2128)
+        * [Add the `fadvise64` system call (dummy implementation)](https://github.com/asterinas/asterinas/pull/2125)
+        * [Add the `ioprio_get` and `ioprio_set` system calls (dummy implementation)](https://github.com/asterinas/asterinas/pull/2126)
+    * Sockets and network:
+        * [Add the `epoll_pwait2` system call](https://github.com/asterinas/asterinas/pull/2123)
+* Enhanced system calls or features:
+    * Processes:
+        * [Add `FUTEX_WAKE_OP` support for the `futex` system call](https://github.com/asterinas/asterinas/pull/2146)
+        * [Add `WSTOPPED` and `WCONTINUED` support to the `wait4` and `waitpid` system calls](https://github.com/asterinas/asterinas/pull/2166)
+        * [Add more fields in `/proc/*/stat` and `/proc/*/status`](https://github.com/asterinas/asterinas/pull/2215)
+    * File systems and I/O in general:
+        * [Add a few more features for the `statx` system call](https://github.com/asterinas/asterinas/pull/2127)
+        * [Fix partial writes and reads in writev and readv](https://github.com/asterinas/asterinas/pull/2230)
+        * [Introduce `FsType` and `FsRegistry`](https://github.com/asterinas/asterinas/pull/2267)
+    * Sockets and network:
+        * [Support some missing socket options for UNIX stream sockets](https://github.com/asterinas/asterinas/pull/2192)
+        * [Add `NETLINK_KOBJECT_UEVENT` support for netlink sockets](https://github.com/asterinas/asterinas/pull/2109)
+        * [Truncate netlink messages when the user-space buffer is full](https://github.com/asterinas/asterinas/pull/2155)
+        * [Enable UNIX sockets to send and receive file descriptors](https://github.com/asterinas/asterinas/pull/2268) 
+        * [Fix the networking address reusing behavior (`SO_REUSEADDR`)](https://github.com/asterinas/asterinas/pull/2277)
+    * Security:
+        * [Add basic cgroupfs implementation](https://github.com/asterinas/asterinas/pull/2121)
```

Comment:
Why cgroupfs belongs to `Security` instead of `File systems and I/O in general`?